bundles/org.openhab.binding.tesla/README.md

```diff
@@ -182,9 +182,13 @@ Additionally, these advanced channels are available (not all are available on al
 | shiftstate                | String                   | Shift State                   | Indicates the state of the transmission, “P”, “D”, “R”, or “N”                                                   |
 | sidemirrorheaters         | Switch                   | Side Mirror Heaters           | Indicates if the side mirror heaters are switched on                                                             |
 | smartpreconditioning      | Switch                   | Smart Preconditioning         | Indicates if smart preconditioning is switched on                                                                |
+| softwareupdateavailable   | Switch                   | Update Available              | Car Software update available, automatically generated comparing "version" to "update version"                   |
```

Comment:
To me this looks like a duplicate channel as `softwareupdateversion` channel only holds a value if it is different, to the existing version.

Reply:
Yes, it is more of a ease-of-life channel to be able to create simpler rules without lots of string comparisons.